### ISSUE TITLE
Data Packs are considered "mods" on the app & frontend even though there is a separate listing for them.

### ISSUE THREAD

**Issue by KTrain5169** [modrinth/code#3073]
### Please confirm the following.

- [x] I checked the [existing issues](https://github.com/modrinth/code/issues) for duplicate problems
- [x] I have tried resolving the issue using the [support portal](https://support.modrinth.com)

### What browsers are you seeing the problem on?

Microsoft Edge, Chrome (including Arc, Brave, Opera, Vivaldi), Firefox, Safari

### Describe the bug

The website does not filter out datapacks from the mods listing.

### Steps to reproduce

1. Go to https://modrinth.com or open the Modrinth App
2. Click on the Discover content > Mods button (mr.com)/Click on the compass (Discover) button and select Mods (MR App)
3. Search for a datapack (An example: [Terralith](https://modrinth.com/mods?q=Terralith))
4. The datapack will appear in the mods section.

### Expected behavior

The datapack does not appear in the search listing.

### Additional context

I'm not sure if this is a true bug since Modrinth does package it into a mod .jar, therefore *technically* making it a mod. However, I was informed by the support agent that this is a bug on the frontend and advised me to file a report.

**Comment by malucard** [comment]
That seems to be fully intentional on the dev's part. If you click the GitHub badge, Terralith says:

> If you would like the mod version, you can download it from [Modrinth](https://modrinth.com/mod/terralith).

The impression I've had is rather that data pack devs often package them as mods for convenience, rather than Modrinth doing that.

**Comment by KTrain5169** [comment]
Nope, Modrinth has an option to package datapacks as mods automatically. In this case I suppose the author has taken it into account though.

**Comment by Erb3** [comment]
I'd say this is a feature, only datapacks that are also packaged as mods are found in the search.

**Comment by elhertz** [comment]
Packaged datapacks should have their own tag. 
The mods page is full of packaged datapacks. I don't want datapacks. I want mods coded in java. Let me filter them out please!

**Comment by IdrisQe** [comment]
https://github.com/modrinth/code/issues/3177 should probably be merged with this, as they're for essentially the same thing.

Also once again bringing attention to this. Surely it isn't that hard to add a filter for the already-existing "Data Pack" category in the Mods section? _Please?_
![Image](https://github.com/user-attachments/assets/0ed19474-6d7d-4d85-a5c4-d0d4b3163355)

**Comment by AspieGamer13** [comment]
My experience shows that ONLY Data Packs that ALSO have a Mod download are the only things showing up under the Mod search.

**Comment by pabienko** [comment]
Is this issue gonna be resolved or is it stays like this? 
It's like a fucking headache trying to find mods under literally hundreds of data packs, that have the same name, but can't function properly with other mods... cause they are data packs. Why would data packs should be listed in the same category as standalone mods?
